capa/v28.1.1/release.diff

```diff
@@ -0,0 +1,149 @@
+  - name: k8s-audit-metrics                                     |      name: irsa-servicemonitors
+                                                                >      version: 0.0.1
+                                                                >    - dependsOn:
+                                                                >      - kyverno
+                                                                >      name: k8s-audit-metrics
+    version: 0.9.0                                                     version: 0.9.0
+    dependsOn:                                                  |    - dependsOn:
+    - kyverno                                                          - kyverno
+  - name: k8s-dns-node-cache                                    |      name: k8s-dns-node-cache
+    version: 2.6.2                                                     version: 2.6.2
+    dependsOn:                                                  |    - dependsOn:
+    - kyverno                                                          - kyverno
+  - name: metrics-server                                        |      name: metrics-server
+    version: 2.4.2                                                     version: 2.4.2
+    dependsOn:                                                  |    - dependsOn:
+    - kyverno                                                   <
+  - name: net-exporter                                          <
+    version: 1.19.0                                             <
+    dependsOn:                                                  <
+    - prometheus-operator-crd                                          - prometheus-operator-crd
+  - name: network-policies                                      |      name: net-exporter
+    version: 0.1.1                                              <
+    catalog: cluster                                            <
+    dependsOn:                                                  <
+    - cilium                                                    <
+  - name: node-exporter                                         <
+    version: 1.19.0                                                    version: 1.19.0
+                                                                >    - catalog: cluster
+    dependsOn:                                                         dependsOn:
+                                                                >      - cilium
+                                                                >      name: network-policies
+                                                                >      version: 0.1.1
+                                                                >    - dependsOn:
+    - kyverno                                                          - kyverno
+  - name: observability-bundle                                  |      name: node-exporter
+    version: 1.3.4                                              |      version: 1.19.0
```

Comment:
```suggestion
    version: 1.6.1                                              |      version: 1.19.0
```

Reply:
What is the reason to upgrade the observability bundle in this release?

This is a patch release created to fix the upgrade issue between v28 and v29. Changes related to other non critical issues should not be included in this release and should only go into [v29.1.0](https://github.com/giantswarm/releases/pull/1348), which is already the case.

Reply:
I would like to upgrade observability-bundle to be able to start using Alloy as our new observability agent in order to get feedback from real world scenarios on the recent work we did in Atlas.
This is non-critical and can wait a v29.1.0